src/main/java/edu/umich/ctools/sectionsUtilityTool/Utils.java

```diff
@@ -249,4 +281,46 @@ public static HttpResponse executeApiCall(HttpUriRequest clientRequest) {
 		return response;
 	}
 
+	public static boolean didApiReturnedWithOutErrors(String errMsg, ApiResultWrapper arw) {
+		if (arw.getStatus() != HttpStatus.SC_OK) {
+			M_log.error(errMsg);
+			return false;
+		}
+		return true;
+	}
+
+	public static String getLTICustomParam(HttpServletRequest request, String ltiParam){
+		HashMap<String, Object> customValuesMap = getLTICustomParams(request);
+		for (String LtiParamKey : customValuesMap.keySet()) {
```

Comment:
Can return null if coustomValuesMap.containsKey(ltiParam) is false; otherwise, use coustom.get(ltiParam) to get the value.